internal/controller/operator/converter/v1alpha1/apis_test.go

```diff
@@ -16,19 +16,24 @@ import (
 )
 
 func TestConvertAlertmanagerConfig(t *testing.T) {
-	f := func(name string, promCfg *promv1alpha1.AlertmanagerConfig, validate func(convertedAMCfg *vmv1beta1.VMAlertmanagerConfig) error) {
```

Comment:
Why change this test, it's already f-test pattern, no? And I prefer listing all the parameters if can than having an extra opts struct.

Reply:
with separate struct for test params it's obvious for each case which entity represents each parameter

Reply:
`internal/controller/operator/converter` package is not in diff already